### DISCUSSION TITLE
About Google Client ID setting

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#257] [Q&A]
What can be done by setting the following items in the .env file?

GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=
GOOGLE_CALLBACK_URL=/oauth/google/callback

**Comment by danny-avila** [comment]
It allows social login with Google. Look into client/.env.example as well as the guide in documents directory for guidance

**Reply by tttll1091** [reply]
thanks
